src/loaders/express.js

```diff
@@ -8,6 +8,13 @@ const apiRoutes = require("../api");
 const config = require("../config/env");
 const { dbHandler } = require("../api/middleware/errorHandler");
 
+const helmet = require("helmet");
+
+const RateLimit = require("express-rate-limit");
+const MongoStore = require("rate-limit-mongo");
+
+const API_REQUEST_TIME_WINDOW_MS = 15 * 60 * 1000; // 15 minutes
+const API_MAX_REQUESTS_PER_WINDOW = 100;
```

Comment:
Yeah, I don't know if those values are well thought, they are per IP, not in total, I think I got those from some default I found, but they can be changed

Reply:
Okay, I guess we can roll with this for now and later on see if changes are necessary :slightly_smiling_face:

Reply:
Going back to this, I think we should increase this a bit. Especially given the idea to have infinite scrolling on the search page. Scrolling quickly with a lot of entries might easily go over 100 requests in 15 minutes, given that the person might refresh the page and then go back to browsing for a bit.
However, like you said at the time - these values are heavily based on speculation, since we don't have real user data for now. As such we probably can just go with any value and adapt later. I just think it might be better to overshoot than undershoot 😛

Reply:
I honestly think this is a good starting point since for the search it will probably be only 4-5 reloads (We won't have 1000 active offers in the near future - I hope we reach it someday, though :P)

I don't oppose changing the limit, it's just important to have one, really. But I don't think this will cause us problems for now.

Reply:
Resolving this for now though. Maybe we should create an issue to further analyze and update this later, though.